src/libstd/sync/mpsc/mod.rs

```diff
@@ -272,6 +272,7 @@ use mem;
 use cell::UnsafeCell;
 use marker::Reflect;
 
+#[stable(feature = "rust1", since = "1.0.0")]
```

Comment:
Can this be marked as unstable? These aren't intended to be stable names in the module.